src/mds/main.py

```diff
@@ -95,5 +105,14 @@ def get_version():
 
 @router.get("/_status")
 async def get_status():
+    """
+    Returns the status of all the cached commons. There are two fields per common:
+     * error: if there was no error this will be "none"
+     * last_update: timestamp of the last data pull from the commons
+     * count: number of entries
+    :return:
+    """
+    # return await redis_cache.get_status()
```

Comment:
Not sure how best to represent status when there are two data sources. We probably deal with this for other apps that talk to something like ES + Postgres, but not sure what the best approach is